frontend/src/app/pages/exercises/exercise/shared/utility/parse-csv.ts

```diff
@@ -233,6 +233,19 @@ function getColor(colorString: string): ColorCategory {
     if (color === null) {
         throw new Error('Wrong color format in SK entries');
     }
+    switch (color[0]) {
+        case 'grün':
+            return 'green';
+        case 'gelb':
+            return 'yellow';
+        case 'rot':
+            return 'red';
+        case 'schwarz':
+            return 'black';
+        case 'blau':
+            return 'blue';
+        default:
```

Comment:
Maybe throw something here? Or use `as 'blau' | 'gelb' | 'grün' | 'rot' | 'schwarz'` in the `switch (color[0])` line?  
But don't have an empty `default` and then just return something you know will never match. Probably `as` is the better solution. @Dassderdie Any thoughts on this?

Reply:
Instead of a switch statement you should use an object literal as a map.
This should then result in something like:
```ts
colorTranslationMap[colors[0]]!;
```